Sources/ProjectDescription/TargetDependency.swift

```diff
@@ -105,6 +111,8 @@ public enum TargetDependency: Codable, Equatable {
 
 extension SDKStatus: Codable {}
 
+extension SDKSource: Codable {}
```

Comment:
I'd move the conformance to Coddle to the definition of the enum. I don't see the reason to keep this in a separate line. If we move it to the declaration of the enum we can see at a glance that SDKSource conforms the protocol. I'd also do the same with the `SDKStatus` enum.